core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.directive.html

```diff
@@ -30,7 +30,7 @@
       </a>
     </td>
     <td>
-      <span ng-click="deleteCanonicalStory(summary.getId())" class="fas fa-trash-alt list-summary"> </span>
+      <span ng-click="deleteCanonicalStory(summary.getId())" class="fas fa-trash-alt list-summary delete-icon"> </span>
```

Comment:
Can we include this span tag under anchor tag (like it's done in other places in the table (line 18, 23))?

(Asking this for accessibility reasons i.e, to have anchor or button tag for clicks.)

Reply:
Done